src/main/java/com/jfrog/ide/common/persistency/ScanCache.java

```diff
@@ -92,14 +92,14 @@ void setScanCacheMap(ScanCacheMap scanCacheMap) {
         this.scanCacheMap = scanCacheMap;
     }
 
-    private void addComponents(Map<String, ? extends Component> components, Severity severity, String summary, String packageType) {
+    private void addComponents(Map<String, ? extends Component> components, Severity severity, String summary, String packageType, List<? extends Cve> cves) {
+        String cveId = ListUtils.emptyIfNull(cves).stream().map(Cve::getId).filter(StringUtils::isNotBlank).findAny().orElse("");
```

Comment:
Let's add here a comment, explaining that although a component may multiple CSVs, we're searching for the first one which is not empty. Let's also explain why we're doing this.